exodus_core/analysis/static_analysis.py

```diff
@@ -340,30 +348,30 @@ def save_icon(self, path):
         icon = self.get_icon_path()
         if icon is None:
             return None
-
         try:
             with zipfile.ZipFile(self.apk_path) as z:
                 with open(path, 'wb') as f:
                     f.write(z.read(icon))
                 _ = Image.open(path)
                 return path
         except:
-            logging.warning('Unable to get the icon from the APK - downloading from details')
+            logging.warning('Unable to get the icon from apk files downloading from details'.format(path))
```

Comment:
The "format" part does not work because there is nothing to replace in the original string.
I would personally keep the original line.

Reply:
I approve your idea.